ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/TxStatsUnitTest.java

```diff
@@ -68,6 +75,13 @@ public void test() throws Exception
         for (int i = 0; i < 100; i++) {
             final AtomicAction D = new AtomicAction();
             D.begin();
+
+            // Let's add some resources
+            D.add(new StatsResource());
```

Comment:
Given this part of the test is also testing part of JBTM-3899 (and not just JBTM-2643) I think the comment just before the for loop could be expanded to clarify that - WDYT @jmfinelli?

Reply:
Or maybe we don't need this bit of the test given you added another test for what I think might be the same thing in the block below where you have ` A.add(new StatsResource());` twice and the assertion??

Reply:
I'll remove the comment

Reply:
Personally I would just delete the reference to https://issues.jboss.org/browse/JBTM-2643 since it was a bug fix.

Reply:
Yeah, I agree with Michael